src/Pages/Rockets.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+import { getApiRockets } from '../Redux/rockets/Rocket';
+import store from '../Redux/configureStore';
+import RocketsList from '../Components/MyRockets/RocketsList';
+
+let flag = false;
+const Rockets = () => {
+  useEffect(() => {
+    if (!flag) {
+      store.dispatch(getApiRockets());
+      flag = true;
+    }
+  });
+  return (
+    <RocketsList />
+  );
+};
```

Comment:
- [Optional]The use of the flag variable is not necessary, as useEffect only runs on the first render of the component. You could simplify the code by removing the flag and using the `useEffect` hook directly: :wink:  `useEffect(() => {
  store.dispatch(getApiRockets());
}, []);
` but all still okay :open_hands: